addons/mass_mailing_sms/static/tests/tours/mass_mailing_activities_split_test_tour.js

```diff
@@ -0,0 +1,45 @@
+/** @odoo-module */
+
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add('mass_mailing_activities_split_test_tour', {
+    test: true,
+    url: '/web',
+    steps: [
+        {
+            content: 'Open Activity Systray',
+            trigger: '.o-mail-ActivityMenu-counter',
+        }, {
+            content: 'Open Email Activities',
+            trigger: '.o-mail-ActivityGroup[data-model_name="mailing.mailing"] span:contains("Email Marketing")',
+        }, {
+            content: 'Open Email Marketing record in the kanban view',
+            trigger: '.o_kanban_record:contains("New Email!")',
+            run: () => {
+                const kanbanCards = document.querySelectorAll('.o_kanban_record .o_kanban_record_headings h3 span');
+                for (let i = 0; i < kanbanCards.length; ++i) {
+                    if (kanbanCards[i].innerText == "New SMS!") {
+                        console.error('SMS Marketing record should not appear in this view');
+                    }
+                }
+            },
+        }, {
+            content: 'Open Activity Systray',
+            trigger: '.o-mail-ActivityMenu-counter',
+        }, {
+            content: 'Open SMS Activities',
+            trigger: '.o-mail-ActivityGroup[data-model_name="mailing.mailing"] span:contains("SMS Marketing")',
```

Comment:
Is `[data-model_name="mailing.mailing"]` really that useful? `:contains(SMS/Email Marketing)` seems good enough, and it's visible on UI whereas `data-model_name` is not.
Removing this improves readability of test ;)